### ISSUE TITLE
[BUG] - `nuxt-quasar` type is overriding every property of Nuxt instance type

### ISSUE THREAD

**Issue by son-nguyen-301** [Maiquu/nuxt-quasar#101]
### Checks

- [X] I have searched the existing issues for a similar bug and it does not exist!

### Environment

Working directory: web-app                                                                         10:20:08 AM
Nuxt project info:                                                                                          10:20:08 AM

------------------------------
- Operating System: Windows_NT
- Node Version:     v20.16.0
- Nuxt Version:     3.13.1
- CLI Version:      3.13.1
- Nitro Version:    2.9.7
- Package Manager:  yarn@1.22.22
- Builder:          -
- User Config:      compatibilityDate, devtools, modules, ssr, eslint, i18n, tailwindcss
- Runtime Modules:  @nuxt/eslint@0.4.0, @nuxtjs/i18n@8.5.3, @nuxtjs/tailwindcss@6.12.1, nuxt-quasar-ui@2.1.4
- Build Modules:    -
------------------------------

                                                                                                                                                            10:20:08 AM  
👉 Report an issue: https://github.com/nuxt/nuxt/issues/new?template=bug-report.yml
👉 Suggest an improvement: https://github.com/nuxt/nuxt/discussions/new
👉 Read documentation: https://nuxt.com

### Quasar version

v2.16.11

### Browsers

Microsoft Edge

### Description

![image](https://github.com/user-attachments/assets/8a8122b9-9854-4e02-9b94-5114b4477cf6)

As you can see in the screenshot. Nothing related to Nuxt instance is being suggested.

Original Nuxt instance property:
![image](https://github.com/user-attachments/assets/cd63a9e3-83fd-4c13-a3a8-dc6b289240a7)

### Reproduction steps

```bash
1. Add `nuxt-quasar` plugin
2. Run dev server
4. See error
```

### Screenshots

```bash
![DESCRIPTION](LINK.png)
```

**Comment by Maiquu** [comment]
It's most likely an upstream issue. `quasar` augmenting `@vue/runtime-core` module might be breaking things, since most suggestions you used in your nuxt environment augments `vue`.

Vue documentation also [suggests](https://vuejs.org/guide/typescript/options-api.html#augmenting-global-properties) augmenting `vue` to modify global properties. There is not much I can do in this module. You should open an issue/PR at [quasar](https://github.com/quasarframework/quasar).

**Comment by Maiquu** [comment]
Probably related: https://github.com/quasarframework/quasar/issues/17416

**Comment by Maiquu** [comment]
This should be fixed with quasar 2.17.0.
